.env

```diff
@@ -0,0 +1,3 @@
+REACT_APP_ACCESS_KEY = hmKgsw9NWcBpwl6zT8C2RQwxhtNPUHsLI35_MhPNlbM
+
+REACT_APP_WEATHER_KEY = 5e61cbd191757aaebdee1b7a03357e99
```

Comment:
Should've added this to git ignore.

Reply:
Actually I have done this, I cant figure out what the problem is, Ill look at it again